### ISSUE TITLE
Replace strftime with something that doesn't need the document object

### ISSUE THREAD

**Issue by stewartknapman** [mattmccray/liquid.js#45]
@mattmccray would you consider replacing strftime with something that doesn't need the document object?

The reason I ask, is I am working on a static generator that is running in the command line and the document object doesn't exist in this environment. I'm going to implement the change in my fork but I wanted to know if it was something you would be interested in?

The libs I'm considering are https://github.com/samsonjs/strftime and https://github.com/moment/moment

**Comment by mattmccray** [comment]
Yeah, I'm all for that. I'd go with strftime, moment too heavy for this use IMO.

**Comment by stewartknapman** [comment]
Yea, that was my ferling as well. I'll let you know when I have something.

**Comment by stewartknapman** [comment]
Question: The tests seem to be passing for me without `etc/split.js`, and I can't see where it is being used. 
How necessary is that?

**Comment by mattmccray** [comment]
If I recall correctly, that was for older IE browsers. I'm not sure if it's really necessary anymore or not. I'd say if it passes on IE9+, then it's good.

IE8 is dead, yay!
